Lib/_aix_support.py

```diff
@@ -0,0 +1,77 @@
+"""Shared AIX support functions."""
+
+import sys
+import subprocess
+from sysconfig import get_config_var
+
+"""
+AIX filesets are identified by four decimal values: V.R.M.F.
+V (version) and R (release) can be retreived using ``uname``
+M(aintenance) and F(ix) values are not available via uname
+For determining ABI compatibility the values V.R.M
+are essential - suplemented by a final value known as
+`builddate` that is expressed as YYWW (Year WeekofYear)
+
+The values for V.R.M of the system Python was built on are available in
+the variable sysconfig.get_config_var("BUILD_GNU_TYPE"). The system builddate
+value may be available in sys.config.get_config_var("AIX_BUILDDATE")
+When all of these values on a running system are greater or equal to the
+values of the build system then the bdist modules are binary compatible
+with the AIX kernel.
+
+For pep425 purposes the AIX platform tag becomes:
+"AIX-{}-{:04d}-{}".format(vmtl, builddate, bitsize)
+e.g., "AIX-6107-1415-32" for AIX 6.1 TL7 bd 1415, 32-bit
+and, "AIX-6107-1415-64" for AIX 6.1 TL7 bd 1415, 64-bit
+"""
+
+# _bd - set to impossible setting year98-week98 if None
+_tmp = str(get_config_var("AIX_BUILDDATE"))
+_bd = 9898 if (_tmp == "None") else int(_tmp)
+_bgt = get_config_var("BUILD_GNU_TYPE")
+_is_32bit = sys.maxsize == 2147483647
+
+
+def _aix_tag(vrtl, bd):
+    # type: (int, int) -> str
+    sz = 32 if _is_32bit else 64
```

Comment:
This could as well be set on the module level, like other similar vars.

Reply:
Now that I understand - a bit better - how pytest works I could consider this. There are so many ways people have approached this - my head spins!

Reply:
How is this related to pytest?

Reply:
Not directly related to pytest of course. However, it took me several days of trying to figure out how pytest and converge worked. Had to modify many ideas to find a way to get the code to pass pypa/packaging. 

Since I made a mess of things with a rebase of my first PR I just redid it, but used the code developed for ``pypa/pip`` and ``pyp/packaging`` - for the levels of Python that do not have this backported,

I could look for something already imported for _is_32bit rather than declare it again. But this is least resistance.

Reply:
Done.